examples/async-sign/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this empty file for?

Reply:
Will be full filled in the next commit(s)